src/exasol_odbc/dialect.tdd

```diff
@@ -952,16 +964,16 @@
       <formula part='year'>CASE WHEN %2 IS NULL OR %3 IS NULL THEN CAST(NULL AS INTEGER) ELSE (EXTRACT(YEAR FROM %3) - EXTRACT(YEAR FROM %2)) END</formula>
       <formula part='quarter'>FLOOR(MONTHS_BETWEEN(TRUNC( CAST(%3 AS DATE), &apos;Q&apos; ), TRUNC( CAST(%2 AS DATE), &apos;Q&apos; )) / 3)</formula>
       <formula part='month'>FLOOR(MONTHS_BETWEEN(TRUNC( CAST(%3 AS DATE), &apos;MONTH&apos; ), TRUNC( CAST(TO_DATE(%2, &apos;YYYY-MM-DD&apos;) AS DATE), &apos;MONTH&apos; )))</formula>
-      <formula part='week'>ABS(FLOOR((DAYS_BETWEEN(%3, &apos;1995-01-01&apos;) - MOD(7 + TO_NUMBER(TO_CHAR(%3, &apos;D&apos;))- 1, 7) - DAYS_BETWEEN(%2, &apos;1995-01-01&apos;) + MOD(7 + TO_NUMBER(TO_CHAR(%2,&apos;D&apos;))- 1, 7))/7))</formula>
+      <formula part='week'>FLOOR((DAYS_BETWEEN(%3, &apos;1995-01-01&apos;) - MOD(7 + TO_NUMBER(TO_CHAR(%3, &apos;D&apos;))- 1, 7) - DAYS_BETWEEN(%2, &apos;1995-01-01&apos;) + MOD(7 + TO_NUMBER(TO_CHAR(%2,&apos;D&apos;))- 1, 7))/7)</formula>

```

Comment:
Bold move removing the `ABS`. This breaks existing behavior. Please check with the Tableau team, if they expect only positive numbers here.

Reply:
But this is what their tests expect. After I removed ABS, the tests passed

Reply:
According to the tests expectations, the don't expect only positive numbers, but both positive and negative. We can double  check with the Tableau team anyway.